SConstruct

```diff
@@ -218,7 +210,12 @@ replacements = {
     '@uintmax_t_decl@': "",
     '@uintptr_t_decl@': "",
     '@off_t_decl@' : 'typedef int64_t wt_off_t;',
-    '@wiredtiger_includes_decl@': wiredtiger_includes
+    '@HAVE_SYS_TYPES_H_DECL@' : '#include <sys/types.h>',
+    '@HAVE_INTTYPES_H_DECL@': "",
```

Comment:
Add a comment to describe this odd-out line?

Reply:
Yep, good point! I've added a comment :)